core/bin/external_node/src/config/mod.rs

```diff
@@ -237,13 +244,12 @@ pub struct OptionalENConfig {
     /// 0 means that sealing is synchronous; this is mostly useful for performance comparison, testing etc.
     #[serde(default = "OptionalENConfig::default_miniblock_seal_queue_capacity")]
     pub miniblock_seal_queue_capacity: usize,
-    /// Polling period for mempool cache update - how often the mempool cache is updated from the database.
-    /// In milliseconds. Default is 50 milliseconds.
-    #[serde(default = "OptionalENConfig::default_mempool_cache_update_interval")]
-    pub mempool_cache_update_interval: u64,
-    /// Maximum number of transactions to be stored in the mempool cache. Default is 10000.
-    #[serde(default = "OptionalENConfig::default_mempool_cache_size")]
-    pub mempool_cache_size: usize,
+    /// Configures whether to persist protective reads when persisting L1 batches in the state keeper.
+    /// Protective reads are never required by full nodes so far, not until such a node runs a full Merkle tree
+    /// (presumably, to participate in L1 batch proving).
+    /// By default, set to `true` as a temporary safety measure.
+    #[serde(default = "OptionalENConfig::default_protective_reads_persistence_enabled")]
+    pub protective_reads_persistence_enabled: bool,
```

Comment:
Side-comment: we probably should revisit the EN docs to better describe possibilities for configuration. Doesn't have to be in this PR.

Reply:
Yep, I think it makes sense to do separately. There's [a tracking task for this in Linear](https://linear.app/matterlabs/issue/PLA-850/update-en-docs).